docs/pages/versions/unversioned/sdk/webbrowser.md

```diff
@@ -66,24 +67,72 @@ Returns a Promise:
+
+#### Arguments
+
+- **package (_string_)** -- **Optional** -- package of browser to be warmed up. If not set, preferred browser will be warmed.
+
+#### Returns
+
+The promise resolves with `{ type: warming, package: string }`
+
+### `WebBrowser.mayInitWithUrl(url, package)`
+
+_Andrdoi Only_
+
+This method initiates (if needed) [CustomTabsSession](https://developer.android.com/reference/android/support/customtabs/CustomTabsSession.html#maylaunchurl) and calls its `mayLaunchUrl` method for browser specified by the package.
+
+#### Arguments
+
+- **url (_string_)** -- url of page that usert is expected to load
+- **package (_string_)** -- **Optional** -- package of browser to be informed. If not set, preferred browser will be used.
+
+#### Returns
+
+### `WebBrowser.coolDown(package)`
+
+_Andrdoi Only_
+
+This methods removes all bindings to services created by warmUp or mayInitWithUrl. You should call this method once you don't need them to avoid potential memory leaks. However, those binding would be cleared once your application is destroyed, which might be sufficient in most cases.
+
+#### Arguments
+
+- **package (_string_)** -- **Optional** -- package of browser to be cooled. If not set, preferred browser will be used.
+
+#### Returns
+
+The promise resolves with `{ result: 'cooling' }` when cooling is performed, or `{ result: 'Nothing to cool down' }` when there was no connection to be dismissed.
 
 ### `WebBrowser.dismissBrowser()`
 
-Dismisses the system's presented web browser. On Android calling this method doesn't actually do anything, the user still has to manually dismiss the browser.
+_iOS Only_
+
+Dismisses the system's presented web browser.
 
 #### Returns
 
 The promise resolves with `{ type: 'dismiss' }`.
 
 ### `WebBrowser.getCustomTabsSupportingBrowsers`
 
-Available only on Android.
+_Android only_
 
-Returns two lists of applications package names supporting Custom Tabs. Under `packages` key you can find a list of all the applications. Under `default` key only the default applications will be listed. Note that list might not be fully reliable, as it uses `PackageManager.getResolvingActivities`.
+Returns list of applications package names supporting Custom Tabs, Custom Tabs service, user chosen and preffered one. This may not be fully reliable, because ot follows `PackageManager.getResolvingActivities` behavior with its flaws. For example, some browsers might not be visible on packages list after other browser is set to dafult.
 
 #### Returns
 
-The promise resolves with `{ packages: string[], default: string[] }`
+The promise resolves with `{ packages: string[], default: string, service: string[], preferred: string }`
+
+- **packages (_string[]_)** : All packages recognized by PackageManager as capable of handling Custom Tabs. Empty array means there is no supporting browsers on device.
```

Comment:
```suggestion
- **browserPackages (_string[]_)** : All packages recognized by `PackageManager` as capable of handling Custom Tabs. Empty array means there is no supporting browsers on device.
```

how about renaming to ^?  + packagemanager into backticks as code entity 🙂 

Would you also write some paragraph describing what are those tabs and services so that a regular expo user who doesn't need to know anything about custom tabs is able to use this feature easily?